extensions/ql-vscode/src/model-editor/model-evaluator.ts

```diff
@@ -75,7 +76,7 @@ export class ModelEvaluator extends DisposableObject {
         ),
       {
         title: "Run model evaluation",
-        cancellable: false,
+        cancellable: true,
```

Comment:
I'm not sure we want this to be cancelable because we have a button on the view to stop the run. @shati-patel?

Reply:
Ah yes. Apologies I spotted this in the code but didn't realise it was being changed in this PR.

We can make it cancellable via the progress bar (which would be nice because that's how users normally expect to be able to cancel things) if we use `MultiCancellationToken` on line 75 above.

When calling `this.runVariantAnalysis`, use `new MultiCancellationToken(token, this.cancellationSource.token)` (and add `token` as an arg to the `withProgress` callback).

Reply:
I've opened https://github.com/github/vscode-codeql/pull/3464 which would hopefully have caught this, and also helped fine one other alert in our codebase.

Reply:
Thanks! I'll change this to use the `MultiCancellationToken` since it probably makes sense to have a cancel button on a notification that we're showing. If we only want the cancel button in the model editor view, we should probably also only show progress in the model editor view.

Reply:
👀 Would it be okay to leave this as non-cancellable for now? 🙏🏽 

I agree that it makes sense for it to be cancellable in future, but we need to do a bit more testing to see that the interactions between the UI buttons and the pop-up work correctly!

Reply:
Yes, sure, I've set `cancellable: false` for now and removed the `MultiCancellationToken` creation.

Reply:
Thank you! ✨